storybook/stories/Layout/AppText.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { storiesOf } from "@storybook/react-native";
+import { AppText } from "../../../src/components/Layout/AppText";
+import { View, Text } from "react-native";
+import { size, color } from "../../../src/common/styles";
+
+const appTextElement = (): JSX.Element[] => {
+  return [
+    <>
+      <AppText>
+        <Text> Default app text</Text>
+      </AppText>
+    </>,
+    <>
+      <AppText style={{ color: color("red", 60) }}>
+        <Text> customize app text</Text>
```

Comment:
Would it be better if you put the style value as addition to the text? Like `customized app text (color("red", 60))`? wdyt?

Reply:
updated